script.js

```diff
@@ -0,0 +1,49 @@
+// Testing on one cell clicking to change to
+// X and adding another class to cell
```

Comment:
Excellent way to get started! Looping through dom elements to check the innerHTML will be the same here as it is on line 46 below.